uasc/symmetric_security_header.go

```diff
@@ -35,9 +35,14 @@ func (h *SymmetricSecurityHeader) Encode() ([]byte, error) {
 }
 
 // String returns Header in string.
-func (s *SymmetricSecurityHeader) String() string {
+func (h *SymmetricSecurityHeader) String() string {
 	return fmt.Sprintf(
 		"TokenID: %d",
-		s.TokenID,
+		h.TokenID,
 	)
 }
+
+// Len returns the Header Length in bytes.
+func (h *SymmetricSecurityHeader) Len() int {
```

Comment:
Do we still need the `Len()` methods? I've removed them from all other structs.

Reply:
Yes; I’ll need these two for the crypto to figure out which bytes to encrypt and which to leave plaintext. 
The message header and the (A)symmetricSecurityHeaders stay plaintext and the rest is encrypted so I need a way to find that barrier in the encoded byte stream. 
Obviously the Asymmertic one is more important as it’s variable length but having both makes the code easier to understand.